core/domain/feedback_services_test.py

```diff
@@ -86,6 +86,15 @@ def test_get_exp_id_from_thread_id(self):
         self.assertEqual(
             feedback_services.get_exp_id_from_thread_id(thread_id), 'exp1')
 
+    def test_get_message_count(self):
+        exp_id = '0'
+        feedback_services.create_thread(
+            'exploration', exp_id, None, 'a subject', 'some text')
+        threadlist = feedback_services.get_all_threads(
+            'exploration', exp_id, False)
+        thread_id = threadlist[0].id
+        self.assertEqual(feedback_services.get_message_count(thread_id), 1)
```

Comment:
Can we also add a message, and test that count is 2?

Reply:
Okay !